shared/test/test_deliverer.rb

```diff
@@ -0,0 +1,48 @@
+require_relative 'test_helper'
+require 'cdo/poste'
+
+class FakeSmtp
+  def send_message(message, from_address, *to_addresses)
+    @message = message
+    @from_address = from_address
+    @to_addresses = to_addresses
+  end
+end
+
+class DelivererTest < Minitest::Test
+  def setup
+    @fake_smtp = FakeSmtp.new
+    Deliverer.any_instance.stubs(:reset_connection).returns(@fake_smtp)
+    @deliverer = Deliverer.new({})
```

Comment:
Great clean setup.

Reply:
thanks for pointing me in this direction!